docs/documentation/changelog.md

```diff
@@ -6,14 +6,11 @@ nav: ../
 ---
 
 # History of Changes
+{% comment %}There are no new lines after "for" and after "endfor" below in order to keep formatting of the output list sane.{% endcomment %}
 * [Introduction and explanation of symbols](#introduction)
-* [Version 42.1.1 (2017-05-05)](#version_42.1.1)
-	* [Contributors to this release](#contributors_42.1.1)
-* [Version 42.1.0 (2017-05-04)](#version_42.1.0)
-	* [Contributors to this release](#contributors_42.1.0)
-* [Version 42.0.0 (2017-02-18)](#version_42.0.0)
-	* [Contributors to this release](#contributors_42.0.0)
-* [Version 9.4.1212 (2016-11-01)](#version_9.4.1212)
+{% for post in site.categories.new_release %}* [Version {{ post.version }} ({{ post.date | date: "%Y-%d-%m"}})](#version_{{ post.version }})
```

Comment:
Didn't catch this earlier :smile: but we must stick to ISO 8601, date: "%Y-%m-%d"